wolfssl/internal.h

```diff
@@ -3052,6 +3057,12 @@ struct WOLFSSL_CTX {
         CallbackRsaEnc    RsaEncCb;     /* User Rsa Public Encrypt  handler */
         CallbackRsaDec    RsaDecCb;     /* User Rsa Private Decrypt handler */
     #endif /* NO_RSA */
+    CallbackGenPreMaster        GenPreMasterCb;     /* Use generate pre-master handler */
+    CallbackGenMasterSecret     GenMasterCb;        /* Use generate master secret handler */
+    CallbackGenSessionKey       GenSessionKeyCb;    /* Use generate session key handler */
+    CallbackSetKeys             SetKeysCb;          /* Use set keys handler */
+    CallbackTlsFinished         TlsFinishedCb;      /* Use Tls finished handler */
+    CallbackVerifymac           VerifymacCb;        /* Use Verify mac handler */
```

Comment:
Maintain camel case, should be `VerifyMacCb`

Reply:
Changed